setup.py

```diff
@@ -113,7 +113,7 @@ def get_cmdclass():
     install_requires=[
         'numpy',
         'scikit-learn',
-        'pandas',
+        'pandas<2',
```

Comment:
I think this means `>=2.0,<2.1`. This will install pandas 2, which is breaking. That won't fix the issue.
And I think the syntax is `~2.0`.
My suggestion would be

- fix breakage by limiting the version (this PR),
- open an issue about upgrading pandas version to 2. 
- work on that separately

Reply:
Sorry, I made a mistake: I meant `'pandas~=1.0',`:) 

`~=` is the entire major series. It's a bit confusing sometimes. 
https://peps.python.org/pep-0440/#examples

Reply:
That's true, I was thinking about poetry's dependencies. Let me update this.